book/03-hacking-atom/sections/02-init-file.asc

```diff
@@ -0,0 +1,25 @@
+=== The Init File
+
+When Atom finishes loading, it will evaluate `init.coffee` in your `~/.atom` directory, giving you a chance to run arbitrary personal CoffeeScript code to make customizations. You have full access to Atom's API from code in this file. If customizations become extensive, consider creating a package, which we will cover in <<_package_word_count>>.
```

Comment:
Is the "personal" in `arbitrary personal CoffeeScript code` needed here (or alternatively, "arbitrary")

And how about `Code in this file has full access to https://atom.io/docs/api/latest/Atom[Atom's API].`